src/server/game/AI/SmartScripts/SmartAI.cpp

```diff
@@ -594,7 +594,7 @@ void SmartAI::AttackStart(Unit* who)
 
     if (who && me->Attack(who, mCanAutoAttack))
     {
-        me->GetMotionMaster()->Clear(MOTION_PRIORITY_NORMAL);
+        //me->GetMotionMaster()->Clear(MOTION_PRIORITY_NORMAL);
```

Comment:
should we do something about this in case mCanCombatMove is false ? maybe that's the only case when to clear the motion generator ?